store/postgres/migrations/2020-12-7-190800_subgraph_error_changes/up.sql

```diff
@@ -0,0 +1,9 @@
+alter table
+    subgraphs.subgraph_error
+drop
+    column block_number;
+
+alter table
+    subgraphs.subgraph_error
+add
+    column created_at timestamp NOT NULL DEFAULT (now() at time zone 'utc');
```

Comment:
The data type should be `timstamptz` so that [the timezone gets stored](https://www.postgresql.org/docs/9.6/datatype-datetime.html), too. Just `timestamp` defaults to `without time zone`. And I think you can just say `default now()` if we use a type with time zone.

Reply:
Done.